service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/AllocationDecisionControllerAboutToStartTest.java

```diff
@@ -58,6 +58,7 @@ void shouldAddYesToMissingAllocationDecision() throws Exception {
         AllocationDecision expectedDecision = AllocationDecision.builder()
             .proposal("test")
             .proposalReason("decision reason")
+            .judgeLevelRadio("No")
```

Comment:
I am a bit nervous we added quite complex logic to the aboutToStart but no extra assertions, would be great if you could add two / three extra scenario.

Reply:
added three extra tests :)